packages/rs-drive-abci/src/common/helpers/mod.rs

```diff
@@ -1,6 +1,8 @@
 pub mod fee_pools;
 pub mod setup;
 
+// TODO: Remove Decimals entirely?
+
 #[cfg(test)]
 mod tests {
```

Comment:
Move test to appropriate place if we even need it

Reply:
This isn't part of this PR.

Reply:
Let's leave todo to do not forget